### ISSUE TITLE
Polymer 3

### ISSUE THREAD

**Issue by rwestlund** [bendavis78/paper-time-picker#63]
This needs to be run through the modulizer and pushed to npm.

**Comment by rwestlund** [comment]
Heads up:
Chrome started showing a deprecation warning that HTML imports will be removed in March 2019.
https://www.chromestatus.com/feature/5144752345317376

**Comment by admwx7** [comment]
TL;DR
* Jumping straight to lit-element (Polymer 4)
* Early alpha coming beginning of January (hopefully)
* Work has started on the new elements, you can see it here: https://github.com/admwx7/am-time-picker
* Ownership will be moved under https://github.com/CommunityElements

The long of it, the modularizer script came back with thousands of errors in this repo, after some investigation the effort to resolve was quite high.  That combined with the known tech debt and the fact it will have to get reworked to support something like lit-html anyway, it makes more sense to just jump into that now.  I can also start utilizing some things I've been working with lately such as TS to get some better docs and such.

I've been collaborating with a few other members in the polymer community for a while now and we've been tossing around the idea of an org on github to supplement element support and drive better quality standards.  We created a channel on the Polymer slack and I will be moving the new element under that org on github.  The goal of this is to ensure this element will continue to get fixes and support even when I get swamped with life.

I've gotten a start on the project with a temporary naming and location (https://github.com/admwx7/am-time-picker) once it gets to an alpha stage I'll rename it and move it under the org I mentioned above.  I have a month of vacation for the holidays starting this coming Friday, aside from a few days of travel my goal is to hit an early alpha before January.
